treeherder/model/models.py

```diff
@@ -189,7 +189,8 @@ class Bugscache(models.Model):
     crash_signature = models.TextField(blank=True)
     keywords = models.TextField(blank=True)
     os = models.CharField(max_length=64, blank=True)
-    modified = models.DateTimeField()
+    modified = models.DateTimeField(db_index=True)
+    whiteboard = models.CharField(max_length=255)
```

Comment:
Yeah good idea to add this. I wonder what the max field length for whiteboard is in Bugzilla?

Reply:
How would I find that info? I don't see a public bugzilla repo.

Reply:
I'd ask :dylan on IRC

Reply:
eseyman said the form element has max 40 char so that should be OK to set the field length to that.